packages/stryker/src/ScoreResultCalculator.ts

```diff
@@ -0,0 +1,125 @@
+import { MutantResult, MutantStatus, ScoreResult } from 'stryker-api/report';
+import { freezeRecursively } from './utils/objectUtils';
+import * as _ from 'lodash';
+import * as path from 'path';
+
+export default class ScoreResultCalculator {
```

Comment:
As we're not renaming the `ScoreResult` to `MutationScoreResult`, i would suggest leaving the name as it is.